services/auth/token/lambdas/authorize.js

```diff
@@ -0,0 +1,19 @@
+import to from 'await-to-js';
+
+import { verifyToken } from '../helpers/token';
+import generateIAMPolicy from '../helpers/generateIAMPolicy';
+
+export const main = async event => {
```

Comment:
main = async event => {...
main = async (event) => {...
according to my linting rules